Makefile

```diff
@@ -7,29 +7,27 @@ APP_DIR ?= /app
 PORT ?= 9080
 DOCKER_RUN_ARGS ?= -v ${MOUNT_DIR}\:${APP_DIR} -w ${APP_DIR}
 DOCKER_PORT_ARGS ?= -p "${PORT}:${PORT}"
-TEST_RUN_ARGS ?=
-TEST_RUN_TIMEOUT ?= 10000
 
 run:
 	docker run ${DOCKER_RUN_ARGS} ${DOCKER_PORT_ARGS} ${IMAGE} node run.js
 
-test:
-	docker run ${DOCKER_RUN_ARGS} ${IMAGE} \
-	    /node_modules/.bin/mocha --timeout=${TEST_RUN_TIMEOUT} ${TEST_RUN_ARGS} tests
+local-tests:
+	npm run test
+	npm run lint
+	npm run lint-json
 
-test-macros:
+test:
 	docker run ${DOCKER_RUN_ARGS} ${IMAGE} \
-	    /node_modules/.bin/mocha --timeout=${TEST_RUN_TIMEOUT} ${TEST_RUN_ARGS} tests/macros
+	  /node_modules/.bin/jest -w1
 
 lint:
 	docker run ${DOCKER_RUN_ARGS} ${IMAGE} \
-	    /node_modules/.bin/jshint --show-non-errors lib tests
+	  /node_modules/.bin/eslint *.js src tests
 
-lint-macros:
+lint-json:
 	docker run ${DOCKER_RUN_ARGS} ${IMAGE} \
-	    /node_modules/.bin/ejslint "macros/**/*.ejs"
-	docker run ${DOCKER_RUN_ARGS} ${IMAGE} \
-	    /node_modules/.bin/jsonlint-cli "macros/**/*.json"
+	  /node_modules/.bin/jsonlint -q *.json `find src macros tests -name '*.json'`
```

Comment:
This works:

```
$ find . -name '*.json' -not -path './node_modules/*' -print0 | xargs -n1 -0 ./node_modules/.bin/jsonlint -q
```
Unfortunately, it doesn't print the filename of the file with an error when it fails, so it is sub-optimal. Weird that there is not an obviously better tool on npm. Lots of options for actually validating against a schema, but not an obvious command-line tool for looking for syntax errors.

Reply:
This is a brand-new repo, but it looks like it would do what we want: lint all json files, ignoring the directories in .gitignore: https://www.npmjs.com/package/json-linter-2

Reply:
Fixed by commit b200111